tools/integration_tests/list_large_dir/testdata/upload_twelve_thousand_files_to_bucket.sh

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
```

Comment:
can we rename it to upload_files_to_bucket.sh ? because this script is not doing anything special particularly to upload "12000" files.